vlmd-submission-tools/pyproject.toml

```diff
@@ -20,18 +20,15 @@ vlmd-submission-tools = 'vlmd_submission_tools.__main__:main'
 python = "^3.9"
 requests = "^2.27.1"
 PyYAML = "^6.0"
-urllib3 = "1.26.9"
-boto3 = "^1.26.122"
+urllib3 = ">=1.26.9"
+heal = {git = "https://github.com/uc-cdis/heal-platform-sdk.git", rev = "master"}
```

Comment:
Can change to PyPI after heal-sdk is deployed

Reply:
I think we need a re-lock?

Reply:
Yes, correct! I'm re-locking to pull in the latest tag of heal-sdk, 1.0.1. I will run a few tests and then will commit the updated pyproject.toml and poetry.lock

Reply:
New lock file has been committed.